Demo/HYPFormsTests/HYPFormsCollectionViewDataSourceTests.m

```diff
@@ -82,6 +82,48 @@ - (void)testIndexInForms
     [self.dataSource processTarget:[HYPFormTarget showFieldTargetWithID:@"last_name"]];
 }
 
+- (void)testEnableAndDisableTargets
+{
+    HYPFormField *targetField = [self.manager fieldWithID:@"display_name" includingHiddenFields:YES];
+
+    XCTAssertFalse(targetField.disabled);
+
+    HYPFormTarget *disableTarget = [HYPFormTarget disableFieldTargetWithID:@"display_name"];
+
+    [self.dataSource processTarget:disableTarget];
+
+    XCTAssertTrue(targetField.disabled);
+
+    HYPFormTarget *enableTarget = [HYPFormTarget enableFieldTargetWithID:@"display_name"];
+
+    [self.dataSource processTargets:@[enableTarget]];
+
+    XCTAssertFalse(targetField.disabled);
+}
+
+- (void)testUpdatingTargetValue
+{
+    HYPFormField *targetField = [self.manager fieldWithID:@"display_name" includingHiddenFields:YES];
+    XCTAssertEqualObjects(targetField.fieldValue, @" ");
+
+    HYPFormTarget *updateTarget = [[HYPFormTarget alloc] initWithDictionary:@{
+                                                                              @"id" : @"display_name",
+                                                                              @"type" : @"field",
+                                                                              @"action" : @"update",
+                                                                              @"target_value": @"John Hyperseed"}];
```

Comment:
Targets now support setting values to targets.
So if it has a value when you push and update, it will use that target_value on that field.
Try running the Sample and switch username, then you can see it in action.